aip_xx1_launch/launch/pointcloud_preprocessor.launch.py

```diff
@@ -57,13 +57,19 @@ def launch_setup(context, *args, **kwargs):
         extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
     )
 
+    glog_component = ComposableNode(
+        package="glog_component",
+        plugin="GlogComponent",
+        name="glog_component",
+    )
+
     # set container to run all required components in the same process
     container = ComposableNodeContainer(
         name=LaunchConfiguration("container_name"),
         namespace="",
         package="rclcpp_components",
         executable=LaunchConfiguration("container_executable"),
-        composable_node_descriptions=[],
+        composable_node_descriptions=[glog_component],
```

Comment:
![image](https://github.com/tier4/aip_launcher/assets/94814556/cf962994-b3cb-4fbb-badb-ce52037aafca)
I found that the container including `glog_component` wasn't executed.
The reason is the`LaunchConfiguration("use_pointcloud_container")=True` so only `concat_loader` will be executed. I think you should add `glog_component` into `concat_loader` instead. 

```
    # load concat or passthrough filter
    concat_loader = LoadComposableNodes(
        composable_node_descriptions=[glog_component, concat_component],
        target_container=target_container,
        condition=IfCondition(LaunchConfiguration("use_concat_filter")),
    )
```

```suggestion
        composable_node_descriptions=[],
```

Reply:
Thanks, I add glog_component to concat_loader :bow: 

https://github.com/tier4/aip_launcher/commit/9d8ed6a225839b747ca20d5d36d04b0dc8f911bb